torch/utils/backend_registration.py

```diff
@@ -77,7 +85,9 @@ def rename_privateuse1_backend(backend_name: str) -> None:
         # to implement torch.ones.
         >>> a = torch.ones(2, device="foo")
         """
-    return _rename_privateuse1_backend(backend_name)
+    _rename_privateuse1_backend(backend_name)
+    global _privateuse1_backend_name
+    _privateuse1_backend_name = backend_name
```

Comment:
@albanD  for a second look here

Reply:
Sounds ok to me.
Can we open an issue to track this @heidongxianhua please?
This will most likely involve teaching dynamo about `_get_privateuse1_backend_name` which shouldn't be too hard for someone that knows dynamo well.

Reply:
> Sounds ok to me. Can we open an issue to track this @heidongxianhua please? This will most likely involve teaching dynamo about `_get_privateuse1_backend_name` which shouldn't be too hard for someone that knows dynamo well.

yeah, I will give an issue later. And could you approve it again to merge this? @janeyx99